lib/utils.js

```diff
@@ -161,11 +161,20 @@ module.exports.mapToMac = function (options) {
 module.exports.actionJackerDecorator = function (emitter, options, fn, mapper) {
   options = clone(options);
   fn = fn || function (err, data) {};
-  return function (err, data) {
-    fn.apply(emitter, [err, data]);
-    if (err || !mapper || !data) return;
-
-    var key = mapper(data);
+  return function (err, data) {
+
+    var resultantData = data;
+    // Sanitize the data
+    if(resultantData) {
+      resultantData = resultantData.toLowerCase().trim();
+      if(resultantData.match(/^activate/)) {

```

Comment:
Good call, doesn't hurt to add it in. We use `.toLowerCase()` above so it shouldn't ever affect it.